frontend/app/components/modal/modal-dialog.tpl.html

```diff
@@ -0,0 +1,15 @@
+<div class="modal fade" data-ng-attr-id="{{ modalID }}">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal"
+            aria-label="Close"><span aria-hidden="true">&times;</span>
```

Comment:
instead of 

`<span aria-hidden="true">&times;</span>`

use

`<span class="fa fa-times"></span>`